site/docs/v1/tech/apis/identity-providers/_samlv2-response-body.adoc

```diff
@@ -112,9 +128,15 @@ ifdef::samlv2_idp_initiated[]
 This field will be `SAMLv2IdPInitiated`.
 endif::[]
 
+[field]#identityProvider.uniqueIdClaim# [type]#[String]#::
+FusionAuth will use this claim as a unique and unchanging value from the external IdP when linking a user.
```

Comment:
Same comment as in the above.

Reply:
Yes, this is part of the Idp initiated flow and shows up there.